### PR TITLE
Remove passport from solana

### PR DESCRIPTION
**Pull request by alireza-sharifpour**
#3553

### CODE REVIEW

File: pages/passport.tsx [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+import { useGeneralWallet } from '@/providers/generalWalletProvider';
+import { ChainType } from '@/types/config';
+import ErrorsIndex from '@/components/views/Errors/ErrorsIndex';
 
 const PassportRoute = () => {
 	const { formatMessage } = useIntl();
+	const { walletChainType } = useGeneralWallet();
+
+	if (walletChainType === ChainType.SOLANA) {
+		return <ErrorsIndex statusCode='404' />;
```
Comment (aminlatifi) [inline pages/passport.tsx:13]: @alireza-sharifpour Please communicate this with PMs.

**Review by aminlatifi — APPROVED** [review]

**Review by MohammadPCh — APPROVED** [review]
@alireza-sharifpour Thanks, LGTM. The only remaining thing is the passport page for the Solana users.

**Review by RamRamez — APPROVED** [review]
Thanks @alireza-sharifpour

**Comment by alireza-sharifpour** [comment]
> @alireza-sharifpour Thanks, LGTM. The only remaining thing is the passport page for the Solana users.

I'm waiting for @MoeNick and @divine-comedian on discord channel.

**Pull request merged**